projects/rocfft/clients/tests/CMakeLists.txt

```
@@ -219,17 +219,14 @@ if( BUILD_CLIENTS_TESTS_OPENMP )
   )
 endif()
 
+find_package( GTest QUIET )
 
-
-include( ../cmake/build-gtest.cmake )
-
-if( BUILD_GTEST OR NOT GTEST_FOUND )
+if( GTest_FOUND )
+    target_link_libraries( rocfft-test PRIVATE GTest::gtest )
+else()
```

```suggestion
else()
  include( ../cmake/build-gtest.cmake )
```
I think you need to bring back the above include if gtest wasn't found but haven't looked into `build-gtest.cmake` myself.